src/rdm/responder.c

```diff
@@ -182,6 +182,144 @@ static int rdm_simple_response_cb(dmx_port_t dmx_num,
   return RDM_RESPONSE_TYPE_ACK;
 }
 
+static int rdm_supported_params_response_cb(dmx_port_t dmx_num,
+                                            const rdm_header_t *header, void *pd,
+                                            uint8_t *pdl_out, void *param,
+                                            const rdm_pid_description_t *desc,
+                                            const char *param_str) 
+{
+  // Return early if the sub-device is out of range
+  if (header->sub_device != RDM_SUB_DEVICE_ROOT) {
+    *pdl_out = rdm_pd_emplace_word(pd, RDM_NR_SUB_DEVICE_OUT_OF_RANGE);
+    return RDM_RESPONSE_TYPE_NACK_REASON;
+  }
+
+  if (header->cc != RDM_CC_GET_COMMAND) {
+    //the supported params list is read-only
+      ESP_LOGE(TAG, "RDM_PID_SUPPORTED_PARAMETERS is read-only");
```

Comment:
I don't think this log message is necessary. The RDM controller is the one generating the bad RDM request, not the user. So we shouldn't need to alert the user.

Reply:
agreed